features/support/env.rb

```diff
@@ -29,6 +29,10 @@
   BushSlicer::World.new
 end
 
+def world
+  return BushSlicer::World.new
+end
```

Comment:
you need to create a new world? Why not call `self` to get current world? btw note that between scenario executions, world may disappear or be inaccessible or something like that I remember. I was dealing with formatting. You can see in `Log` class that we fall back to `Kernel` when World does not exist.